src/app/data-model/Nullable.h

```diff
@@ -49,6 +49,10 @@ struct Nullable : protected Optional<T>
     // Optional.
     using Optional<T>::Value;
 
+    // Pull in APIs that make sense on Nullable with the same names as on
+    // Optional.
```

Comment:
you can remove this comment as this is a copy and paste of the above. Just place this right after the `Value` use as it uses the same comment. This the using should be on line 51 without any spacing.

Reply:
Done. PTAL.